redis/src/cluster_async/connections_container.rs

```diff
@@ -112,6 +112,16 @@ where
         }
     }
 
+    /// Returns true if the identifier represents a known primary node.
+    #[allow(dead_code)]
+    pub(crate) fn is_primary(&self, identifier: &Identifier) -> bool {
+        self.connection_for_identifier(identifier).is_some()
```

Comment:
I dont like this approach in which we can return false for a non-existent identifiers. This is a bad design that can hide logical problems in the code. Would not the returning of Option<bool> be more rust-idiomatic ?